blocks/identity-block/features/account-management/_children/PasswordEditableFieldContainer.jsx

```diff
@@ -0,0 +1,161 @@
+import React, { useEffect, useState } from 'react';
+import { useFusionContext } from 'fusion:context';
+import getProperties from 'fusion:properties';
+import getTranslatedPhrases from 'fusion:intl';
+import { useIdentity } from '../../..';
+import EditableFieldPresentational from '../../../components/EditableFormInputField';
+import FormInputField, { FIELD_TYPES } from '../../../components/FormInputField';
+import FormPasswordConfirm from '../../../components/FormPasswordConfirm';
+import passwordValidationMessage from '../../../utils/password-validation-message';
+import validatePasswordPattern from '../../../utils/validate-password-pattern';
+
+function PasswordEditableFieldContainer({ email, hasPassword }) {
+  const [error, setError] = useState(false);
+  const [passwordRequirements, setPasswordRequirements] = useState({
+    status: 'initial',
+  });
+
+  const { arcSite } = useFusionContext();
+  const { locale } = getProperties(arcSite);
+  const phrases = getTranslatedPhrases(locale);
+
+  const { Identity } = useIdentity();
+
+  useEffect(() => {
+    const getConfig = async () => {
+      await Identity.getConfig()
+        .then((response) => {
+          const {
+            pwLowercase,
+            pwMinLength,
+            pwPwNumbers,
+            pwSpecialCharacters,
+            pwUppercase,
+          } = response;
+
+          setPasswordRequirements({
+            pwLowercase,
+            pwMinLength,
+            pwPwNumbers,
+            pwSpecialCharacters,
+            pwUppercase,
+            status: 'success',
+          });
+        })
+        .catch(() => setPasswordRequirements({ status: 'error' }));
+    };
+
+    if (Identity) {
+      getConfig();
+    }
+  }, [Identity]);
+
+  const {
+    pwLowercase = 0,
+    pwMinLength = 0,
+    pwPwNumbers = 0,
+    pwSpecialCharacters = 0,
+    pwUppercase = 0,
+    status,
+  } = passwordRequirements;
+
+  const passwordErrorMessage = passwordValidationMessage({
+    defaultMessage: phrases.t('identity-block.password-requirements'),
+    options: {
+      lowercase: {
```

Comment:
wonder if there's a way to extract these requirements. these are from another place in the repo, right?